README.md

```diff
@@ -18,11 +18,29 @@ SnakeYAML Engine KMP is based on
 
 ## Status
 
-⚠️ SnakeYAML Engine KMP is **under development**.
+The library has been successfully ported to KMP, and is consumed by [kaml](https://github.com/charleskorn/kaml),
+a library that integrates with [kotlinx.serialization](https://github.com/Kotlin/kotlinx.serialization). However,
+it hasn't been battle-tested well enough yet, so please keep it in mind when integrating with critical workflows.
```

Comment:
```suggestion
The library has been successfully ported to KMP, and is consumed by [kaml](https://github.com/charleskorn/kaml),
a library that integrates with [kotlinx.serialization](https://github.com/Kotlin/kotlinx.serialization).
```

snake-kmp is using the YAML test suite, and passes as many tests as snake-java (keeping in mind it's rare for a YAML lib to pass all test-suite cases!) And the tests pass on all Kotlin targets, which I think is a big achievement!

Did you want to communicate something else by "it hasn't been battle-tested"? Could be phrased differently? Otherwise, I propose removing it.

Reply:
I actually didn't think about correctness when writing this (tests do prove what you wrote), but rather various other aspects like performance. We had to replace JDK-specific parts with KMP counterparts, and just this can result in some non-obvious changes with regard to snakeyaml-engine. E. g. See #92, there's some performance regression when switching to the third-party library - some issues may be already present in the currently released version.

Maybe let's write that this KMP port hasn't been widely adopted yet, and is relatively young? Technically we aren't sure about the level of adoption in private projects, but what I want to communicate here is that the maturity of our lib is not equivalent to snakeyaml-engine's. Do you see my point? 😄

Reply:
Yes I think that's a good point regarding performance, I've updated my suggestion - wdyt?